### PR TITLE
Add SHA256 and SHA512

### PR DESCRIPTION
**Pull request by StefanScherer**
I apologize, my C# really isn't perfect, so I just appended the checksum types in a similar way as SHA1.
You may find it useful ;-)

### CODE REVIEW

**Comment by StefanScherer** [comment]
Just tried it with a sample.

```
.\build.bat
curl -k -o o64 https://dl.bintray.com/mitchellh/otto/otto_0.1.0_windows_amd64.zip
.\build_output\checksum\checksum.exe -t sha256 -c 8aa135bd40db9211015626a618b8aaae5487b28eb5fba04a04d5eedfa14fd177 o64
Hashes match.
```

Got the SHA256 from https://dl.bintray.com/mitchellh/otto/otto_0.1.0_SHA256SUMS

File: src/checksum/Program.cs [hunk 1]
```diff
@@ -33,6 +33,14 @@ private static void Main(string[] args)
...
                 hash_util = new SHA1CryptoServiceProvider();
             }
+            else if (configuration.HashType.ToLowerSafe() == "sha256")
+            {
+                hash_util = new SHA256CryptoServiceProvider();
+            }
+            else if (configuration.HashType.ToLowerSafe() == "sha512")
+            {
+                hash_util = new SHA512CryptoServiceProvider();
+            }
```
Comment (ferventcoder) [inline src/checksum/Program.cs]: Probably fine for now. :shipit:

**Comment by ferventcoder** [comment]
Add the changes to https://github.com/ferventcoder/checksum/blob/master/nuget/checksum.nuspec#L12 as well?

**Comment by ferventcoder** [comment]
I'll handle those.

**Comment by StefanScherer** [comment]
@ferventcoder ok, I've pushed a little too slow ;-)

**Comment by StefanScherer** [comment]
That was my last commit https://github.com/StefanScherer/checksum/commit/a452c2a7b1a3aadec24c44983a00614517715fdb

**Comment by ferventcoder** [comment]
grabbed it

**Comment by ferventcoder** [comment]
https://chocolatey.org/packages/checksum/0.0.3

**Pull request merged**